firefox-ios/Ecosia/L10N/String.swift

```diff
@@ -163,14 +163,6 @@ extension String {
         case openAllLinksAutomatically = "Open all links automatically with Ecosia"
         case beClimateActive = "Be climate active every day while browsing"
         case groupYourImpact = "Group your impact"
-        case getATreeWithEveryFriend = "Get a tree with every friend who joins. They get one too!"
-        case aboutEcosia = "About Ecosia"
-        case seeHowMuchMoney = "See exactly how much money we made, how we spent it, and how many trees we planted."
-        case discoverWhereWe = "Discover where we plant trees, and find out how our tree planting projects across the globe are doing."
-        case learnHowWe = "Learn how we protect your privacy by encrypting your searches, never selling your data to advertisers without your permission and more."
-        case findAnswersTo = "Find answers to popular questions like how Ecosia neutralizes CO2 emissions and what it means to be a social business."
-        case aboutEcosiaCollapseAccessibility = "Interact to collapse content"
-        case aboutEcosiaExpandAccessibility = "Interact to expand content"
```

Comment:
Were those also removed from the `.strings` English file? I believe once removed from the source they will be removed from every language, but we still need to remove from english.